kiosk/package.json

```diff
@@ -11,8 +11,10 @@
     "@types/node": "^16.11.33",
     "@types/react": "^18.0.9",
     "@types/react-dom": "^18.0.3",
+    "qrcode.react": "^3.1.0",
```

Comment:
Did we need to pass these new deps through the tool? double check with @abchatra, this one has a non-mit but looks fully permissive so should be fine

Reply:
Oh, yes, this is something I need to ask him about. Will let you know.

Reply:
This has ISC license, which is near identical to MIT. We should add the license to 3rd party notices file and merge for now.